src/account/Memory.ts

```diff
@@ -31,28 +27,29 @@ export function getBufferToSign(
 export default class AccountMemory extends AccountBase {
   override readonly address: Encoded.AccountAddress;
 
+  readonly #secretKeyDecoded: Uint8Array;
```

Comment:
Private properties the same secure as using WeakMap

> In prototypal inheritance, its behavior may be emulated with WeakMap objects

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties